test/authenticator.spec.ts

```diff
@@ -307,15 +283,11 @@ describe('Decentraland Crypto', function () {
       }
       try {
         await ECDSA_PERSONAL_EPHEMERAL_VALIDATOR(authority, authLink, {
-          provider: new HttpProvider(
-            'https://mainnet.infura.io/v3/640777fe168f4b0091c93726b4f0463a'
-          ),
+          provider: new HTTPProvider('https://rpc.decentraland.org/mainnet'),
```

Comment:
Can we use https://rpc.decentraland.org/mainnet?project=dcl-crypto-test so we can discriminate the usage?